src/components/elements/TimePicker.js

```diff
@@ -70,21 +69,24 @@ class TimeAndDatePicker extends Component {
           }}
                 />
         <FlatButton
-          label='submit'
-          primary
-          onClick={() => this.loadUncorrelatedEvents()}
+          label='Select time range'
+          default
+          onClick={() => this.setStartAndEndTime()}
                 />
       </div>
     )
   }
 
-  loadUncorrelatedEvents () {
-    const { dispatch, filters, history } = this.props
+  setStartAndEndTime () {
+    const { dispatch } = this.props
     const { startDate, startTime, endDate, endTime } = this.state
     let start = (startDate && startTime) ? startDate + 'T' + startTime : null
     let end = (endDate && endTime) ? endDate + 'T' + endTime : null
-    const newFilter = Object.assign({}, filters, { startTime: start, endTime: end })
-    dispatch(filterActions.synchronizeFilters(newFilter, null, null, history))
+    if (!(start && end)) {
+      end = DateTime.local().toISO()
```

Comment:
I see this is using local time. Should this be UTC instead?

Reply:
We should use UTC everywhere but actual display of time.